charger/easee.go

```diff
@@ -363,6 +364,20 @@ func (c *Easee) ProductUpdate(i json.RawMessage) {
 		// startup completed
 		c.once.Do(func() { close(c.done) })
 
+	case easee.ERROR_CODE:
+		newErrorCode := value.(int)
+
+		// ERROR_CODE 2048:  "Overcurrent", Easee App "Überstrom erkannt" / red  LED stripe, OpMode 5
+	 	// [easee ] TRACE ... ProductUpdate : ... ERROR_CODE 2048
+       	// [easee ] TRACE ... ProductUpdate : ... ERROR_STRING [ERROR] Charger: Overcurrent EV=18637 > 17000 mA
+		// Reboot or plug out/in solves this issue.
+    	if c.errorCode != 2048 && newErrorCode == 2048 {
+			c.log.WARN.Printf("Error code 2048. Rebooting the charger!")
+			uri := fmt.Sprintf("%s/chargers/%s/commands/%s", easee.API, c.charger, easee.Reboot)
```

Comment:
How long does it take for the charger to come back online? And what is the behavior of evcc while the charger does not respond?